tests/expectations/rename_items.c

```diff
@@ -0,0 +1,39 @@
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#define capi_constant_abc 10
```

Comment:
This is basically the use case for allowing one rule per item type as well as a default global one. I don't often see lower_snake_case #defines in C, even in a largely lower_snake_case API. This PR already has the infrastructure for that, passing `IdentifierType` in to `ExportConfig::rename`, but hasn't implemented it (yet).